vhd/packer/configure-windows-vhd.ps1

```diff
@@ -124,8 +128,8 @@ function Get-FilesToCacheOnVHD {
             "https://packages.aks.azure.com/csi-proxy/v1.1.3/binaries/csi-proxy-v1.1.3.tar.gz"
         );
         "c:\akse-cache\win-k8s\"      = @(
-            "https://packages.aks.azure.com/kubernetes/v1.28.15/windowszip/v1.28.15-1int.zip",
-            "https://packages.aks.azure.com/kubernetes/v1.29.10/windowszip/v1.29.10-1int.zip"
+            "https://kubernetesartifacts.azureedge.net/kubernetes/v1.29.10/windowszip/v1.29.10-1int.zip",

```

Comment:
I thought we were removing `azureedge.net`

Reply:
Thanks, updated